remo/base/api/views.py

```diff
@@ -0,0 +1,9 @@
+from rest_framework.viewsets import ReadOnlyModelViewSet
+
+
+class CorsEnabledReadOnlyViewset(ReadOnlyModelViewSet):
```

Comment:
Since ``CorsEnabledReadOnlyViewset`` is getting way long and unreadable i suggest we go with``BaseReadOnlyModelViewset``. On top of that we might need to add more deployment specific stuff in the future (eg. caching or deployment specific headers).